data/korath/nanobots.txt

```diff
@@ -13,6 +13,7 @@
 
 ship "Nanobot"
 	attributes
+		category "Game Mechanics"
```

Comment:
```suggestion
		category "Unclassified"
```
The nanobot is, in-universe, a thing (albeit evidently representing a swarm of nanobots rather than a single one) rather than an out-of-universe game mechanic, so unclassified is probably more appropriate.